.github/workflows/deploy.yml

```
@@ -0,0 +1,126 @@
+name: CD
+
+on:
+  push:
+    branches:
+      - main
+      - 'release/*'
+  workflow_dispatch: { }
```

YAMLlint (1.37.1)

[error] too many spaces inside empty braces

(braces)